knowledge_base/Environment-Variables.md

```diff
@@ -0,0 +1,394 @@
+- [AWS_ACCESS_KEY_ID](#aws_access_key_id)
+- [AWS_REGION](#aws_region)
+- [AWS_SECRET_ACCESS_KEY](#aws_secret_access_key)
+- [AXIE_SHARED_SECRET](#axie_shared_secret)
+- [CHAIN_PORT](#chain_port)
+- [CLOUDAMQP_URL](#cloudamqp_url)
+- [COSMOS_GOV_V1](#cosmos_gov_v1)
+- [COSMOS_REGISTRY_API](#cosmos_registry_api)
+- [CW_BOT_KEY](#cw_bot_key)
+- [DATABASE_CLEAN_HOUR](#database_clean_hour)
+- [DATABASE_URI](#database_uri)
+- [DATABASE_URL](#database_url)
+- [DD_AGENT_MAJOR_VERSION](#dd_agent_major_version)
+- [DD_API_KEY](#dd_api_key)
+- [DD_DYNO_HOST](#dd_dyno_host)
+- [DD_ENABLE_HEROKU_POSTGRES](#dd_enable_heroku_postgres)
+- [DD_ENABLE_HEROKU_REDIS](#dd_enable_heroku_redis)
+- [DD_DISABLE_HOST_METRICS](#dd_disable_host_metrics)
+- [DD_LOG_LEVEL](#dd_log_level)
+- [DD_SITE](#dd_site)
+- [DISABLE_CACHE](#disable_cache)
+- [DISCORD_BOT_TOKEN](#discord_bot_token)
+- [DISCORD_BOT_URL](#discord_bot_url)
+- [DISCORD_CLIENT_ID](#discord_client_id)
+- [DISCORD_WEBHOOK_URL_DEV](#discord_webhook_url_dev)
+- [DL_BUILD](#dl_build)
+- [ENFORCE_SESSION_KEYS](#enforce_session_keys)
+- [ETH_ALCHEMY_API_KEY](#eth_alchemy_api_key)
+- [ETH_RPC](#eth_rpc)
+- [ETHERSCAN_JS_API_KEY](#etherscan_js_api_key)
+- [FLAG_COMMUNITY_HOMEPAGE](#flag_community_homepage)
+- [FLAG_PROPOSAL_TEMPLATES](#flag_proposal_templates)
+- [FLAG_SIDEBAR_TOGGLE](#flag_sidebar_toggle)
+- [HEROKU_APP_NAME](#heroku_app_name)
+- [IS_CI](#is_ci)
+- [JWT_SECRET](#jwt_secret)
+- [MAGIC_API_KEY](#magic_api_key)
+- [MAGIC_DEFAULT_CHAIN](#magic_default_chain)
+- [MAGIC_PUBLISHABLE_KEY](#magic_publishable_key)
+- [MAGIC_SUPPORTED_BASES](#magic_supported_bases)
+- [MIXPANEL_DEV_TOKEN](#mixpanel_dev_token)
+- [MIXPANEL_PROD_TOKEN](#mixpanel_prod_token)
+- [NEXT_PUBLIC_RSA_PRIVATE_KEY](#next_public_rsa_private_key)
+- [NEXT_PUBLIC_RSA_PUBLIC_KEY](#next_public_rsa_public_key)
+- [NO_CLIENT](#no_client)
+- [NO_GLOBAL_ACTIVITY_CACHE](#no_global_activity_cache)
+- [NO_PRERENDER](#no_prerender)
+- [NO_SSL](#no_ssl)
+- [NODE_ENV](#node_env)
+- [PGPASSWORD](#pgpassword)
+- [PORT](#port)
+- [PROCFILE](#procfile)
+- [RABBITMQ_API_URI](#rabbitmq_api_uri)
+- [RABBITMQ_URI](#rabbitmq_uri)
+- [REDIS_URL](#redis_url)
+- [ROLLBAR_ENV](#rollbar_env)
+- [ROLLBAR_SERVER_TOKEN](#rollbar_server_token)
+- [RPC_HOST](#rpc_host)
+- [SEND_WEBHOOKS_EMAILS](#send_webhooks_emails)
+- [SENDGRID_API_KEY](#sendgrid_api_key)
+- [SERVER_URL](#server_url)
+- [SESSION_SECRET](#session_secret)
+- [SL_BUILD](#sl_build)
+- [SL_PORT](#sl_port)
+- [SLACK_FEEDBACK_WEBHOOK](#slack_feedback_webhook)
+- [SLACK_WEBHOOK_URL_DEV](#slack_webhook_url_dev)
+- [SNAPSHOT_HUB_URL](#snapshot_hub_url)
+- [TELEGRAM_BOT_TOKEN_DEV](#telegram_bot_token_dev)
+- [TEST_ENV](#test_env)
+- [WITH_PRERENDER](#with_prerender)
+- [ZAPIER_WEBHOOK_URL_DEV](#zapier_webhook_url_dev)
+
+## AIRPLANE_SECRET
+
+We use Airplane.dev tasks for our custom domains. See our [Custom-Domains.md](./knowledge_base/Custom-Domains.md) entry for more info.
```

Comment:
We're aiming to deprecate Airplane -- this may be removed soon. We do not use it for Custom Domains.